deployments/kustomize/overlays/openshift/images/base/app/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: app-deployment
+  annotations:
+    image.openshift.io/triggers: >-
```

Comment:
remove annotation trigger, but make sure it is in the overlay for dev, test and prod